### DISCUSSION TITLE
Expected response code "250" but got code "550", with message "550 You are not allowed to send e-mails as the domain gmail.com"

### DISCUSSION THREAD

**Discussion by frankii91** [symfony/symfony#51019] [Q&A]
I need help :)

Expected response code "250" but got code "550", with message "550 You are not allowed to send e-mails as the domain gmail.com"

When using my own mail server of the same provider as chosting and I send e-mails to myself, however, I want to describe them that the sender is someone else - the real e-mail of the client.

Needed for message archiving - creating a ticket in the message ticket system.
Until this port I used the old version of SWIFT ..... and it worked fine for me there.

$to = 'xxx@moja-domena.pl';
$from = 'xxx@moja-domena.pl';
$sender = 'kowalski@gmail.com';

$mailer = new Mailer($transport);
$email = new Email();
$email->to($to);
$email->subject($subject);
$email->priority(Email::PRIORITY_HIGH);
$email->replyTo($reply_to);

$email->from(new Symfony\Component\Mime\Address($from, $from_name));
if (!is_null($sender)) {
	$email->sender(new Symfony\Component\Mime\Address($sender, $sender_name));
}
$email->returnPath($from);
$email->text($template_txt, 'utf-8');
$mailer->send($email);

**Comment by alaugks** [comment]
The email server (SMTP) does not allow emails to be sent from a domain for which it does not feel responsible. For example, I cannot send an email with a foreign domain as the sender via the email server of the company where I work. This is a feature configured by the administrators to prevent SPAM from being sent. 

The error message "550 You are not allowed to send e-mails as the domain gmail.com" also clearly says that.
